packages/discord.js/src/util/Util.js

```diff
@@ -509,16 +509,6 @@ function cleanCodeBlockContent(text) {
   return text.replaceAll('```', '`\u200b``');
 }
 
-/**
- * Lazily evaluates a callback function
- * @param {Function} cb The callback to lazily evaluate
- * @returns {Function}
- */
-function lazy(cb) {
-  let defaultValue;
-  return () => (defaultValue ??= cb());
-}
```

Comment:
The documentation for this should stay, else it would completely disappear from the documentation, right?

Reply:
I think it's safe to assume most people don't use this function outside of discord.js so having this documented would really just mean we have two different doc comments in two different places.